### PR TITLE
improve comment readability

### PR DESCRIPTION
**Pull request by sauerlandf**
Fix for Issue #15

### CODE REVIEW

File: resources/views/date/gig/listAttendances.blade.php
Comment (nielsranosch) [inline resources/views/date/gig/listAttendances.blade.php]: As little JavaScript as possible in the "contents"-section and definitely no $(document).ready stuff. There is a "js"-section at the end of this file, where this can go and it already has a $(document).ready(...), where you can add your stuff to. Can use a class instead of an id and do something like:
`$('.hideable-comment').click(function() { $(this).toggle() })`

File: resources/assets/sass/app.scss
Comment (nielsranosch) [inline resources/assets/sass/app.scss]: Does something like `.overflowing-cell { white-space:normal; }` not work?

File: resources/views/date/gig/listAttendances.blade.php
Comment (nielsranosch) [inline resources/views/date/gig/listAttendances.blade.php]: I never liked the code-flow with $mark and it looks even worse with these changes. This is a great chance to get rid of the variable $mark and instead read "attending"/"maybe-attending"/"not-attending"/"unanswered" from a variable.
If you don't want to rework this (which is understandable), I'll do it.

**Review by nielsranosch — CHANGES_REQUESTED** [review]
The ideas are good and work well. Currently, the code is not in line with the rest of the project.

File: resources/views/date/gig/listAttendances.blade.php
Comment (sauerlandf) [reply resources/views/date/gig/listAttendances.blade.php]: fixed

File: resources/assets/sass/app.scss
Comment (sauerlandf) [reply resources/assets/sass/app.scss]: override statement is now more specific

File: resources/views/date/gig/listAttendances.blade.php
Comment (sauerlandf) [reply resources/views/date/gig/listAttendances.blade.php]: changed it. I'm waiting for your approval

**Comment by sauerlandf** [comment]
added a few more thigs:
- the main menu button to go to events in calendar view was broken (fixed)
- added a similar overview for rehearsal attendances (see Issue #14 )
- added the number of 'maybe' votes in these overviews (related to Issue #45 )

**Comment by sauerlandf** [comment]
> 
> 
> added a few more thigs:
> 
>     * the main menu button to go to events in calendar view was broken (fixed)
> 
>     * added a similar overview for rehearsal attendances
> 
>     * added the number of 'maybe' votes in these overviews (relating to but not closing Issue #45 )

these overviews should be unified for events in the long run though...

**Comment by sauerlandf** [comment]
Pull request #52 is included here because i forgot how to branch. Will be the last time though ;)

**Comment by sauerlandf** [comment]
replaced by pull request #53

**Comment by nielsranosch** [comment]
Don't worry :-)

**Pull request closed without merging**
